Added absolute path for the Default PAA Trust Store

vijs opened the pull request:
#### Problem
* chip-tool is not working without the "--paa-trust-store-path" argument

#### Change overview
* chip-tool will use the Test Attestation Trust Store if "--paa-trust-store-path" is not passed

#### Testing
* Matter Unit Tests
* Tested Commissioning flow (chip-tool vs chip-all-clusters-app)

Damian-Nordic reviewed (approved):
Isn't it possible to run chip-tool without the store? Is there some instruction on which files need to be bundled with `chip-tool` to keep it working on another host?

bzbarsky-apple commented:
> Isn't it possible to run chip-tool without the store?

It is if we keep using `GetTestAttestationTrustStore`.  It's not clear to me whether that's meant to go away or not.  @vijs ?

tcarmelveilleux commented on examples/chip-tool/BUILD.gn:
```diff
@@ -32,7 +32,12 @@ config("config") {
   ]
 
-  defines = [ "CONFIG_USE_SEPARATE_EVENTLOOP=${config_use_separate_eventloop}" ]
+  default_paa_trust_store_path = rebase_path(chip_root, "")
+
+  defines = [
+    "CONFIG_USE_SEPARATE_EVENTLOOP=${config_use_separate_eventloop}",
+    "DEFAULT_PAA_TRUST_STORE_PATH=\"${default_paa_trust_store_path}/credentials/development/paa-root-certs\"",
+  ]
```
This prevents porting the chip-tool and/or cross-compiling the chip-tool on a better host and then move it to a target

bzbarsky-apple replied:
That's a basic issue with requiring use of a file-based trust store, right?

bzbarsky-apple commented:
/rebase

vijs commented:
I've added a changeset to fall back to TestAttestationTrustStore if the --paa-trust-store-path is not passed. We can continue to work on a path to deprecate the TestAttestationTrustStore and use a default Trust Store after updating the test scripts and test harness to use the PAA Trust store 

Also updated the chip-tool README file:
Trust store will be automatically created using the default TestAttestation PAA. To use a different set of PAAs, pass the path using the optional parameter --paa-trust-store-path while running the built executable. Trusted PAAs are available at credentials/development/paa-root-certs/.

vijs commented:
/rebase

vijs commented:
/rebase
